paths_cli/wizard/pause.py

```diff
@@ -0,0 +1,72 @@
+import time
+import contextlib
+from collections import namedtuple
+
+PauseStyle = namedtuple("PauseStyle", ['short', 'long', 'section'])
+
+PAUSE_STYLES = {
+    'testing': PauseStyle(0.01, 0.03, 0.05),
+    'default': PauseStyle(0.1, 0.5, 0.75),
+    'nopause': PauseStyle(0.0, 0.0, 0.0),
+}
+
+_PAUSE_STYLE = PAUSE_STYLES['default']
+
+
+@contextlib.contextmanager
+def pause_style(style):
+    """Context manager for pause styles.
+
+    Parameters
+    ----------
+    style : :class:`.PauseStyle`
+        pause style to use within the context
+    """
+    old_style = get_pause_style()
+    try:
+        set_pause_style(style)
+        yield
+    finally:
+        set_pause_style(old_style)
+
+
+def get_pause_style():
+    """Get the current pause style"""
+    global _PAUSE_STYLE
```

Comment:
is `global` required here? 
```suggestion
```
(pylint claims:  `Using global for '_PAUSE_STYLE' but no assignment is done (global-variable-not-assigned)`)

Reply:
Yeah, that seems to work. I was surprised to need `global`, but I guess I only needed it for setting.

Reply:
Yeah, for setting it defaults to inner scope, but for returning it first searches the inner scope but then goes to module scope